src/CFamily/CMake/Resources.resx

```diff
@@ -117,17 +117,26 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="BadCMakeSettings" xml:space="preserve">
+    <value>[CompilationDatabaseLocator] Could not parse CMakeSettings.json: {0}</value>
+  </data>
+  <data name="BadCompilationDatabaseFile" xml:space="preserve">
+    <value>[CompilationConfigProvider] Could not parse compilation database: {0}</value>
+  </data>
   <data name="FoundCompilationDatabaseFile" xml:space="preserve">
-    <value>[CompilationDatabaseLocator] Reading compilation database from '{0}</value>
+    <value>[CompilationConfigProvider] Reading compilation database from '{0}</value>
   </data>
   <data name="NoBuildConfigInCMakeSettings" xml:space="preserve">
     <value>[CompilationDatabaseLocator] Specified build configuration '{0}' could not be found at '{1}'</value>
   </data>
   <data name="NoCMakeSettings" xml:space="preserve">
-    <value>[CompilationDatabaseLocator] No {0} was found under {1}, searching for compilation database under default location: {2}</value>
+    <value>[CompilationDatabaseLocator] No {0} was found under {1}, returning default location: {2}</value>
+  </data>
+  <data name="NoCompilationDatabaseEntry" xml:space="preserve">
+    <value>[CompilationConfigProvider] Could not find file '{0}' in the compilation database.</value>
   </data>
   <data name="NoCompilationDatabaseFile" xml:space="preserve">
-    <value>[CompilationDatabaseLocator] Could locate compilation database at '{0}'. Make sure that your project is configured correctly. See https://github.com/SonarSource/sonarlint-visualstudio/wiki for more information.</value>
+    <value>[CompilationConfigProvider] Could locate compilation database at '{0}'. Make sure that your project is configured correctly. See https://github.com/SonarSource/sonarlint-visualstudio/wiki for more information.</value>
```

Comment:
I'd put a line break before the second sentence and indent it slightly to make the "where to get help" text easy to see.

Reply:
Since this is a non-debug message, I don't think it should include the `CompilationConfigProvider` prefix - it's an implementation detail, and isn't going to be meaningful to the end user.

I think it's worth having a user-friendly prefix, though e.g. `[CMake]`